01-ruby-fundamentals/intro-to-ruby-gems.md

```diff
@@ -71,15 +116,86 @@ puts "This is blue text on red".blue.on_red.blink
 puts "This is uncolorized".blue.on_red.uncolorize
 ```
 
-## Activity: Divide & Conquer Ruby Gems
-In pods, research the usage of one of the following gems and create a small example showing at least one of the features of the gem.
+Relatively straight-forward, but neat!
```

Comment:
Let's not use this language, since it makes an assumption that isn't necessarily true for a student.